ktoml-core/src/commonMain/kotlin/com/akuleshov7/ktoml/parsers/TomlParser.kt

```diff
@@ -59,10 +62,24 @@ public value class TomlParser(private val config: TomlInputConfig) {
                 // Parse the inline comment if any
                 val inlineComment = line.trimComment()
 
-                if (line.isTableNode()) {
-                    if (line.isArrayOfTables()) {
+                // append all multiline values to StringBuilder as one line
+                if (multilineValueBuilt.isNotEmpty() || line.isStartOfMultilineValue()) {
```

Comment:
The only question here is: how do we understand the difference between a valid multiline array and the case, when user forgot to add ']' in the end of the statement?

Reply:
Thats a good point! Now I detected it